nvi-commons/src/main/java/no/sikt/nva/nvi/common/db/CandidateDao.java

```diff
@@ -345,8 +345,17 @@ public Builder applicable(boolean applicable) {
                 return this;
             }
 
-            public Builder instanceType(InstanceType instanceType) {
-                this.builderInstanceType = instanceType;
+            //TODO: Should be removed once we have migrated instanceType to String
+            public Builder instanceType(String instanceType) {
+                var enums = Arrays.stream(InstanceType.values()).toList();
+                var instanceTypeEnum = enums.stream()
+                                           .filter(value -> value.toString().equals(instanceType))
+                                           .findFirst();
+                if (instanceTypeEnum.isPresent()) {
+                    this.builderInstanceType = instanceTypeEnum.get().getValue();
+                } else {
+                    this.builderInstanceType = instanceType;
+                }
```

Comment:
This piece should be set back as it was after we have run batchScan in all environments.